Explorer/Assets/DCL/Passport/PassportController.cs

```diff
@@ -701,40 +701,52 @@ private void ShowFriendshipInteraction()
 
             async UniTaskVoid FetchFriendshipStatusAndShowInteractionAsync(CancellationToken ct)
             {
-                // Fetch our own profile since inputData.IsOwnProfile sometimes is wrong
-                Profile? ownProfile = await selfProfile.ProfileAsync(ct);
-
-                // Dont show any interaction for our own user
-                if (ownProfile?.UserId == inputData.UserId) return;
-
-                FriendshipStatus friendshipStatus = await friendService.GetFriendshipStatusAsync(inputData.UserId, ct);
-
-                switch (friendshipStatus)
+                try
                 {
-                    case FriendshipStatus.NONE:
-                        viewInstance!.AddFriendButton.gameObject.SetActive(true);
-                        break;
-                    case FriendshipStatus.FRIEND:
-                        viewInstance!.RemoveFriendButton.gameObject.SetActive(true);
-                        break;
-                    case FriendshipStatus.REQUEST_SENT:
-                        viewInstance!.CancelFriendButton.gameObject.SetActive(true);
-                        break;
-                    case FriendshipStatus.REQUEST_RECEIVED:
-                        viewInstance!.AcceptFriendButton.gameObject.SetActive(true);
-                        break;
-                    case FriendshipStatus.BLOCKED:
-                        viewInstance!.UnblockFriendButton.gameObject.SetActive(true);
-                        break;
+                    // Fetch our own profile since inputData.IsOwnProfile sometimes is wrong
+                    Profile? ownProfile = await selfProfile.ProfileAsync(ct);
+
+                    // Dont show any interaction for our own user
+                    if (ownProfile?.UserId == inputData.UserId) return;
+
+                    FriendshipStatus friendshipStatus = await friendService.GetFriendshipStatusAsync(inputData.UserId, ct);
```

Comment:
Can you please review other calls to `GetFriendshipStatusAsync`? Exceptions everywhere should be handled properly.